hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/AutoRenewConfig.java

```diff
@@ -16,14 +16,15 @@
 
 package com.hedera.node.config.data;
 
+import com.hedera.node.config.NetworkProperty;
 import com.swirlds.config.api.ConfigData;
 import com.swirlds.config.api.ConfigProperty;
 import java.util.Set;
 
 @ConfigData("autoRenew")
 public record AutoRenewConfig(
         //         @ConfigProperty(defaultValue = "") Set<EntityType> targetTypes
-        @ConfigProperty(defaultValue = "CONTRACT,ACCOUNT") Set<String> targetTypes) {
+        @ConfigProperty(defaultValue = "CONTRACT,ACCOUNT") @NetworkProperty Set<String> targetTypes) {
```

Comment:
Don't we want the `defaultValue`'s to match mono-service? (C.f. [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties#L118), default should be `autoRenew.targetTypes=`.)

Reply:
Set defaultValue = "". This change caused a few tests failing. Fixed the tests.